media/js/base/send-to-device.js

```diff
@@ -134,59 +135,76 @@ if (typeof window.Mozilla === 'undefined') {
      * Updates the form fields to include SMS messaging
      */
     SendToDevice.prototype.showSMS = function() {
-        var $label = this.$formFields.find('.form-input-label');
-        this.$form.addClass('sms-country');
-        $label.html($label.data('alt'));
-        this.$input.attr('placeholder', this.$input.data('alt'));
+        var $label = document.querySelector('.form-input-label');
+        this.$form.classList.add('sms-country');
+        $label.innerHTML = $label.dataset.alt;
+        this.$input.setAttribute('placeholder', this.$input.dataset.alt);
         this.smsEnabled = true;
     };
 
     /**
      * Binds form submission and click events
      */
     SendToDevice.prototype.bindEvents = function() {
-        this.$form.on('submit', $.proxy(this.onFormSubmit, this));
-        this.$sendAnotherLink.on('click', $.proxy(this.sendAnother, this));
+        this.eventFormSubmit = this.onFormSubmit.bind(this);
+        this.$form.addEventListener('submit', this.eventFormSubmit);
+
+        if(this.$sendAnotherLink) {
+            // Check that the Element exists, as it's not present in unit test spec file
```

Comment:
If the link isn't present in the unit test mock, I'd perhaps favor adding it there versus adding checks in production code around it?